Modules/Loadable/Segmentations/Widgets/qMRMLSegmentEditorWidget.cxx

```diff
@@ -116,7 +116,7 @@ static const int BINARY_LABELMAP_SCALAR_TYPE = VTK_UNSIGNED_CHAR;
 // static const unsigned char BINARY_LABELMAP_VOXEL_FULL = 1; // unused
 static const unsigned char BINARY_LABELMAP_VOXEL_EMPTY = 0;
 
-static const char NULL_EFFECT_NAME[] = "NULL";
+static const char nullptr_EFFECT_NAME[] = "NULL";
```

Comment:
Seems incorrect

Reply:
Thanks, fixed the regex (updated the wiki)